### PR TITLE
Alerting: Show full preview value in tooltip

### LINKED ISSUE #42324: Improve new alerting preview
**What happened**:

The new alerting value preview is not very readable and gets cut off way too soon making it not very useful.

image

**What you expected to happen**:

We should apply some formatting and format the message so that
1. it does not get cut off so soon
2. it makes it easier to parse the returned values

**How to reproduce it (as minimally and precisely as possible)**:

1. Create a new (Grafana managed) alert
2. Run queries (with multiple series)
3. Define an alert condition
4. Preview alert

**Anything else we need to know?**:

**Environment**:
- Grafana version:
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser:
- Grafana plugins:
- Others:

### PR DESCRIPTION
**Pull request by peterholmberg**
**What this PR does / why we need it**:
Small fix for being able to display the full value in preview alert. This will view will be overhauled post 8.3

**Which issue(s) this PR fixes**:

Fixes #42324

image

### CODE REVIEW

File: public/app/features/alerting/unified/components/rule-editor/PreviewRuleResult.tsx [hunk 1]
```diff
@@ -15,6 +15,17 @@ type Props = {
 export function PreviewRuleResult(props: Props): React.ReactElement | null {
   const { preview } = props;
   const styles = useStyles2(getStyles);
+  const fieldConfig: FieldConfigSource = useMemo(() => {
```
Comment (gillesdemey) [inline public/app/features/alerting/unified/components/rule-editor/PreviewRuleResult.tsx:18]: Do we need a `useMemo` if the values we use inside are static?

File: public/app/features/alerting/unified/components/rule-editor/PreviewRuleResult.tsx [hunk 2]
```diff
@@ -37,7 +48,7 @@ export function PreviewRuleResult(props: Props): React.ReactElement | null {
       </div>
     );
   }
-
+  console.log(data);
```
Comment (gillesdemey) [inline public/app/features/alerting/unified/components/rule-editor/PreviewRuleResult.tsx:51]: I think this one needs to be removed before we can merge :)

**Review by gillesdemey — CHANGES_REQUESTED** [review]
`console.log` and minor nit

File: public/app/features/alerting/unified/components/rule-editor/PreviewRuleResult.tsx (hunk 2 above)
Comment (peterholmberg) [reply public/app/features/alerting/unified/components/rule-editor/PreviewRuleResult.tsx:51]: ops

**Review by gillesdemey — APPROVED** [review]
LGTM ✌️

File: public/app/features/alerting/unified/components/rule-editor/PreviewRuleResult.tsx (hunk 1 above)
Comment (peterholmberg) [reply public/app/features/alerting/unified/components/rule-editor/PreviewRuleResult.tsx]: Hm, maybe not.

**Pull request merged**